app/js/lib/foxbox/services/base.js

```diff
@@ -232,7 +237,7 @@ export default class BaseService extends EventDispatcher {
    * super.teardown() method as well.
    */
   teardown() {
-    for (let watcher of this[p.watchers].values()) {
+    for (const watcher of this[p.watchers].values()) {
```

Comment:
> I think it is a valid use for const.

Right, just double checked with [1] :)

> Now we can make this eslint rule an error rather than a warning.

Good point, I'll make eslint to consider it as an error

[1] https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Statements/for...of